_targets.R

```diff
@@ -7,9 +7,35 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
 p1_targets_list <- list(
+  tar_target(
+    site_data_01427207,
```

Comment:
Per our conventions, we add the file extension to the end of file target names, so these would all be `site_data_XXXXXXXX_csv`